src/backend/integration/src/support/test-driver.ts

```diff
@@ -40,6 +40,14 @@ export const BACKEND_WASM_PATH = resolve(
 );
 
 export class TestDriver {
+  /**
+   * Keeps track of the PIC's total time advanced when using {@link advanceTime}.
+   */
+  public get advancedTimeMs(): number {
```

Comment:
Since it's only used in one place so far, how do you feel about tracking this in the test instead? If we find the need for this in another test, then we could move it back here.